frontend/app/src/components/history/events/HistoryEventsList.vue

```diff
@@ -180,7 +182,7 @@ watch(
                   <v-lazy>
                     <history-event-type
                       :event="item"
-                      :chain="getChain(item.location)"
+                      :chain="getChain(toSnakeCase(item.location))"
```

Comment:
Do we really need it to do for `.location`? the values should not be affected by our conversion.

The only places would be where it is used as the key

Reply:
yes, because it doesn't pick the correct url for the chain on the hashlink component (takes the default which is etherscan)

and in this object, location comes in as **polygon pos**

<img width="276" alt="image" src="https://github.com/rotki/rotki/assets/5625068/54f2c551-169b-4d9b-a6bc-a526a9061268">

Reply:
I did ask on Discord, but Lef said this value is from the enum, so I have to handle on FE

Reply:
think to avoid future issues, we should call it in `getChain`
since, for single name chains (eth, avax, ...) `toSnakeCase` does not affect and combined names will always be snake case...